auth0/v3/authentication/passwordless.py

```diff
@@ -79,8 +79,76 @@ def sms(self, client_id, phone_number, client_secret=None):
             data=data
         )
 
+    def login(self,
```

Comment:
You copy-pasted the "login for database" method signature. For passwordless there are parameters you don't require. The signature should take: `client_id, email, code, scope` where scope would have a default value of "openid".

Please, rename this method to `email_login` to match the existing SMS variant.